### PR TITLE
Enable inspector to be closed and re-opened without loss of notebook and console connectivity.

### PR DESCRIPTION
**Pull request by afshin**
Fixes https://github.com/jupyter/jupyterlab/issues/1038

### CODE REVIEW

File: src/inspector/plugin.ts [hunk 1]
```diff
@@ -22,31 +22,80 @@ import {
...
+    if (!manager.inspector || manager.inspector.isDisposed) {
+      manager.inspector = newInspector();
+      app.shell.addToMainArea(manager.inspector);
+      return;
+    }
+    if (manager.inspector.isAttached) {
+      app.shell.activateMain(manager.inspector.id);
+      return;
     }
+    console.log(manager.inspector);
```
Comment (blink1073) [inline src/inspector/plugin.ts]: extraneous log

**Comment by blink1073** [comment]
Looks good once that log is removed.

File: src/inspector/plugin.ts (hunk 1 above)
Comment (afshin) [inline src/inspector/plugin.ts]: Oh oops, cheers.

**Comment by ellisonbg** [comment]
Looks good...

**Pull request merged**